alacritty/src/renderer/mod.rs

```diff
@@ -175,15 +176,30 @@ impl Renderer {
         size_info: &SizeInfo,
         glyph_cache: &mut GlyphCache,
     ) {
-        let cells = string_chars.enumerate().map(|(i, character)| RenderableCell {
-            point: Point::new(point.line, point.column + i),
-            character,
-            extra: None,
-            flags: Flags::empty(),
-            bg_alpha: 1.0,
-            fg,
-            bg,
-            underline: fg,
+        let mut skip_next = false;
+        let cells = string_chars.enumerate().filter_map(|(i, character)| {
+            if skip_next {
+                skip_next = false;
+                return None;
+            }
```

Comment:
Isn't it easier to just increment i twice for wide chars?

Reply:
But it's a part of enumerate and we need to skip a space? Like the issue is that we have a space after a wide char here.

I decided to avoid rewriting all of the handling to do `(char, width)` like iterators because it was getting really messy.

Reply:
Nobody forces you to use enumerate? Seems simler to roll your own loop.

Reply:
yeah, but you need to skip `space` regardless? Like the point is to skip next character and not advance the `i` with `+2`.